conf.py

```diff
@@ -18,6 +20,38 @@
 ### Extensions
 ############################################################
 
+#############################################################
```

Comment:
Can you move the addition to a new section at the end of the file?

Reply:
Done